From 9f16e1590127e6ab0791eb750c7644477ebf43f7 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey@solus-project.com>
Date: Tue, 28 Mar 2017 22:08:28 +0100
Subject: [PATCH 12/36] postinstall: Revert janky workaround for
 clr-boot-manager

Now we understand the nature of the bug, we can revert this change and
treat it at the source.

This bug is caused by ikeydoherty/clr-boot-manager#54

Signed-off-by: Ikey Doherty <ikey@solus-project.com>
---
 os_installer2/postinstall.py | 12 ------------
 1 file changed, 12 deletions(-)

diff --git a/os_installer2/postinstall.py b/os_installer2/postinstall.py
index ee98ad4..cb05ca1 100644
--- a/os_installer2/postinstall.py
+++ b/os_installer2/postinstall.py
@@ -736,18 +736,6 @@ class PostInstallBootloader(PostInstallStep):
             TODO: In future this will be managed by clr-boot-manager itself,
             we're just ensuring that we write the EFI variables
         """
-        # Tell clr-boot-manager where our ESP is for now
-        try:
-            if not os.path.exists("/dev/disk/by-partlabel"):
-                os.makedirs("/dev/disk/by-partlabel", 00755)
-            if os.path.lexists("/dev/disk/by-partlabel/ESP"):
-                os.remove("/dev/disk/by-partlabel/ESP")
-            os.symlink(self.installer.locate_esp(),
-                       "/dev/disk/by-partlabel/ESP")
-        except Exception as e:
-            self.set_errors("Failed to simulate ESP link: {}".format(e))
-            return False
-
         # Proxy back to CBM
         if not self.apply_boot_loader():
             return False
-- 
2.13.3

